aries-site/src/pages/templates/datatable-customization.mdx

```diff
@@ -20,31 +20,31 @@ import { Box } from 'grommet';
 
 ## Guidelines
 
-For complex tables with many columns, is it important for the user to be able to 
+For complex data tables with many columns, is it important for the user to be able to 
```

Comment:
Should this be `DataTable` ?

Reply:
I think this one can be left as lower case since its more about the general concept